server/src/app/auth/auth.service.spec.ts

```diff
@@ -110,9 +114,9 @@ describe('AuthService', () => {
         instance(mockValidationService));
 
       await expect(service.signUp(address, name, message, signature))
-        .rejects.toThrow();
+        .rejects.toThrow(Error);
```

Comment:
Can u tell me where this error from? 
i cannot find the declaration

Reply:
@inDlife When verify() returns false, it throws error.